mis_builder/models/mis_report.py

```diff
@@ -317,7 +335,7 @@ def name_get(self):
                 name = u'{}.{}'.format(
                     rec.kpi_id.name, rec.subkpi_id.name)
             else:
-                name = rec.kpi_id.name
+                name = rec.kpi_id.display_name
```

Comment:
not sure about this one

Reply:
When a user will create a budget item for example, he will be able to search KPI expressions based on KPI description, but he won't be able to see his search in the results. I think this is confusing.

Reply:
But maybe we could show it depending of a value in the context.

Reply:
What bothers me most is the fact that the name would be structured differently when there is a subkpi.